src/renderer/proccessors/lineMeshProcessor.ts

```diff
@@ -0,0 +1,9 @@
+import { ModelNode, RenderNode } from '~/renderer/common'
```

Comment:
I think the directory name has an extra `c` in it.

Reply:
Lol yup.